server/src/main/java/org/elasticsearch/search/aggregations/metrics/SumAggregatorFactory.java

```diff
@@ -46,6 +50,22 @@
         super(name, config, queryShardContext, parent, subFactoriesBuilder, metaData);
     }
 
+    static void registerAggregators(ValuesSourceRegistry valuesSourceRegistry) {
+        valuesSourceRegistry.register(SumAggregationBuilder.NAME,
+           List.of(CoreValuesSourceType.NUMERIC, CoreValuesSourceType.DATE),
```

Comment:
Should we support booleans here?  I haven't checked Sum specifically, but I'd bet it accepts a boolean field in master.

Reply:
You're right. I am adding booleans too.